### PR TITLE
Add IE, universal time, in the configuration update command message:

### PR DESCRIPTION
**Pull request by YouShengLiu**
* Add TimeZone field in the AMFContext.

https://github.com/free5gc/free5gc/issues/113

### CODE REVIEW

File: internal/gmm/message/build.go [hunk 1]
```diff
@@ -789,12 +789,23 @@ func BuildConfigurationUpdateCommand(ue *context.AmfUe, anType models.AccessType
 		configurationUpdateCommand.ShortNameForNetwork.SetIei(nasMessage.ConfigurationUpdateCommandShortNameForNetworkType)
 	}
 
+	now := time.Now()
+	universalTimeAndLocalTimeZone := nasConvert.UniversalTimeAndLocalTimeZoneToNas(now)
+	universalTimeAndLocalTimeZone.SetIei(nasMessage.ConfigurationUpdateCommandUniversalTimeAndLocalTimeZoneType)
+	configurationUpdateCommand.UniversalTimeAndLocalTimeZone = &universalTimeAndLocalTimeZone
+
 	if ue.TimeZone != "" {
```
Comment (ss920386) [inline internal/gmm/message/build.go:797]: According to 24.501, 8.2.19.9 Local time zone ,
> This IE may be included to assign a new local time zone to the UE.

When is the ue.TimeZone set? Shall we make it like only when `ue.TimeZone != amfSelf.TimeZone` will AMF send configuration update command to update ue's local time zone?
(I'm not quite familiar with the time zone handling between ue and amf, so feel free to discuss if you have different understanding of spec.)

File: internal/gmm/message/build.go [hunk 2]
```diff
@@ -803,6 +814,12 @@ func BuildConfigurationUpdateCommand(ue *context.AmfUe, anType models.AccessType
 		configurationUpdateCommand.NetworkDaylightSavingTime = nasType.
```
Comment (ss920386) [inline internal/gmm/message/build.go:814]: Line 800 to 823 can me merged with the previous line 797 to line 809.

File: internal/gmm/message/build.go (hunk 1 above)
Comment (YouShengLiu) [reply internal/gmm/message/build.go:797]: Actually, I could not find the code that assigns the time zone value to ue.TimeZone, nor could I find the procedure through which the UE sends its time zone information to the network.
In addition, according to TS 22.042, it seems that providing the time information is the responsibility of the network.
Therefore, I think AMF should store the time zone information of the network.

```
if ue.TimeZone != amfSelf.TimeZone {
    ue.TimeZone = amfSelf.TimeZone
    // ...
    /* Set up the time zone part of the configuration update command */
}
```
Is this better?

Comment (ss920386) [reply internal/gmm/message/build.go:797]: Sorry for my late reply. Yes, I think it is better. Thanks.

**Comment by tim-ywliu** [comment]
update nas pkg as below:
```
go get github.com/free5gc/nas@main
go mod tidy
```

**Review by tim-ywliu — APPROVED** [review]

**Pull request merged**